bin/tma_commander.py

```diff
@@ -163,7 +158,7 @@ def get_command_help(self):
 
     def get_argument_names(self, cmd_name):
         CommandClass = self.command_dict[cmd_name]
```

Comment:
Would it be worth adding a docstring here?

Reply:
That is fair. I have added doc strings to all methods and also more information in the class doc string, including this warning:
```
    Warning: this is only intended as a short-term hack.
    It will be deleted once MTMountCsc has been shown to work
    with the Operation Manager.
```